README.md

```diff
@@ -85,10 +76,10 @@ If not present, the PDF report is not generated.
           log-level: info
 ```
 
-A typical workflow which checks the contents of the repository, runs Security Audit on each of the OpenAPI files found in the project and saves the execution file as artifact would look like this:
+A typical workflow which checks the contents of the repository, runs Security Audit on each of the OpenAPI files found in the project and saves the SARIF file as artifact would look like this:
 
 ```yaml
-name: "pr-workflow"
+name: "core-workflow"
 
 # follow standard Code Scanning triggers
```

Comment:
note: here, "Code Scanning" is uppercase, couple of lines down it's lowercase, would be good to be consistent. I recommend lowercase, it's easier to remember, unless you are referring to a name (you could do either here: refer to "code scanning" as a thing, or to GitHub Code Scanning as GitHub's specific feature (assuming they have a feature with that exact name)).